pkg/logservice/client.go

```diff
@@ -95,6 +95,9 @@ var _ Client = (*managedClient)(nil)
 // to the Log Service in parallel, multiple clients should be created and used
 // to do so.
 func NewClient(ctx context.Context, cfg ClientConfig) (Client, error) {
+	if cfg.MaxMessageSize == 0 {
```

Comment:
no default here. morpc has a default value 100MB

Reply:
have removed it.